README.md

```diff
@@ -132,4 +131,11 @@ with a pcap analysis tool.
 * `xxd` decodes the ascii hex bytestream output from the BufferOverrun exception into binary
 * `od` dumps the output to a format [wireshark can read](https://www.wireshark.org/docs/wsug_html_chunked/ChIOImportSection.html)
 * `text2pcap` (wireshark) appends fake ethernet and IP headers to the SMB packet and writes a pcap file to stdout
-* `tshark` (wireshark) decodes the SMB packet and displays full packet details
\ No newline at end of file
+* `tshark` (wireshark) decodes the SMB packet and displays full packet details
+
+---
+
+This project, _pike_, is released under a Simplified BSD License granted by Dell Inc.'s Open Source Project program,
```

Comment:
might want to say `This project, _pike_ and _pike-smb2_, ...` because the repo is called "pike", but the distribution is called "pike-smb2" (pike was already taken)

Reply:
Fixed.